src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -591,7 +501,7 @@ public static void main(String[] args) throws IOException {
         ExtensionsRunner extensionsRunner = new ExtensionsRunner();
 
         // initialize the transport service
-        extensionsRunner.initializeExtensionTransportService(extensionsRunner.getSettings());
+        extensionsRunner.getNetty4Transport.initializeExtensionTransportService(extensionsRunner.getSettings(), extensionsRunner);
```

Comment:
We should avoid passing the object of `extensionRunner` here directly. Instead, all of the required methods, variables should be moved to `Netty4Transport` file.